src/components/App.js

```diff
@@ -1,116 +1,31 @@
 import React from 'react';
-import MarkdownIt from 'markdown-it';
-import MarkdownItMusic from 'markdown-it-music'
-import queryString from 'query-string';
-import { createBrowserHistory } from 'history';
 import { HashRouter as Router, Route, Link } from 'react-router-dom';
+import Markdown from './MarkdownItRender.js';
+import ResponsiveContainer from './ResponsiveContainer.js';
 import './App.scss';
+import '../../node_modules/bootstrap/dist/css/bootstrap.min.css';
 
-const history = createBrowserHistory();
+function App() {
+  return (
+    <ResponsiveContainer childElement={HomeRouter()} />
```

Comment:
Can we put the HomeRouter inline here?

Reply:
I was conflicted. My ultimate reasoning to breaking out is in the future, when we have potentially future routes, that will be defined in `HomeRouter`, whereas `App` should only contain UI elements and no logic.  Let me know if you disagree.